src/components/IOUConfirmationList.js

```diff
@@ -57,7 +57,8 @@ const propTypes = {
         phoneNumber: PropTypes.string,
     })).isRequired,
 
-    /** Is this IOU associated with existing report */
+    /** Is this IOU associated with existing report. This is true when the action is initiated from inside a group chat and it's false when the action is initiated from the
+     * floating-action-button */
     isIOUAttachedToExistingChatReport: PropTypes.bool.isRequired,
```

Comment:
There is a part of me that wants to rename this prop to `canModifyParticipants` and make it more clear what's happening. Let me know if you agree.

Reply:
yeah the name is hard to read and I don't understand what it supposed to be right now, your suggestion makes sense

Reply:
if it's the only thing that it's doing right now